client/me/concierge/info-step.js

```diff
@@ -4,38 +4,129 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
 
 /**
  * Internal dependencies
  */
-import Button from 'components/button';
 import CompactCard from 'components/card/compact';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextarea from 'components/forms/form-textarea';
+import Timezone from 'components/timezone';
 import PrimaryHeader from './primary-header';
 import Site from 'blocks/site';
+import { localize } from 'i18n-calypso';
+import { updateConciergeInfo } from 'state/concierge/info/actions';
+import { getConciergeInfo } from 'state/selectors';
 
 class InfoStep extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			timezone: moment.tz.guess(),
```

Comment:
This is really neat! But can we default instead to the customer's selected WordPress.com account timezone?

Reply:
I tried searching for it, I know that in the WP-Admin there is a site timezone. But could not find anything related to the customer I guess I could grab it from site settings but as far as I know there is no actual form to update it in Calypso.

Reply:
I found two selectors in Calypso: https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-site-timezone-name.js and https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-site-timezone-value.js

I haven't tested these so I'm not sure if they'll work or not. Let's leave this as-is, I filed a new issue to look into this if we have time: 501-gh-hg

Reply:
I used the same one, but it gets the site timezone from settings and like I said I could not really find a way to change that timezone in calypso.